README.md

```diff
@@ -146,7 +146,49 @@ If wanting to supply dynamic suggestions, you will need to provide a function wh
 
 | Name | Type | Description |
 | --- | --- | --- |
-| results | <code>Array.&lt;string&gt;</code> | The results to show in the suggestions menu |
+| options | <code>Array.&lt;*&gt;</code> | The options which match the entered query |
+
+
+### mapOptionToSuggestedValue
+
+This function is used to convert the options returned from the `source` function into a their respective strings to be used as the suggestion values within the suggestions menu.
```

Comment:
Started this suggestion to correct the "into a their" typo, but think it can be simplified a bit, what do you think?
```suggestion
This function is used to convert the options returned from the `source` function into strings for the suggestions menu.
```

Reply:
I like it 👍